### ISSUE TITLE
Show informative text when opening app for the first time

### ISSUE THREAD

**Issue by melvin4u445** [ARK-Builders/ARK-Navigator#105]
As you see in the below screenshots, there is no information on how to get started after installing the app. 

On the Manage folders section, there is a lot of unused white space which could be used to explain how it works. So we could add some text like, "To get started, tap the plus button to add a root." and maybe more text could be added to reduce the learning curve. The Navigate data section could also display some text on how to get started

<image src=https://user-images.githubusercontent.com/17034768/136686623-df32f78f-c788-47bd-b469-3636db0e20e2.png width=40%> <image src=https://user-images.githubusercontent.com/17034768/136686619-8e9471a6-ab05-4603-9aec-098188c9a834.png width=40%>

**Comment by kirillt** [comment]
Good idea. Maybe we even can create "first time guide" consisting of bubbles with instructions and arrows to action items.
